crates/stdarch-gen/src/main.rs

```diff
@@ -1437,15 +1737,21 @@ fn gen_arm(
             link.push_str(&link_aarch64);
             link.replace("_EXT_", ext).replace("_EXT2_", ext2)
         };
+        let abi = if test_fn == "load_test" {
+            "unadjusted"
+        } else {
+            "C"
+        };
```

Comment:
I think we would want to always use `"unadjusted"` when linking with LLVM intrinsics, unless I'm missing something?

Reply:
I'm not sure if these links depend on "C" Abi, especially other architectures, like x86. In view of the fact that most of the structures are marked by `#[repr(simd)]`, the difference in Abi should only affect the structure containing multiple vectors IMO. Let's try it.

Reply:
Using "unadjusted" looks fine, at least for arm/aarch64. It also slightly simplifies the compilation process, so I think it makes sense. 
A few `assert_instr`s of some instructions using `simd_extract` need to be correct, I think this may be due to some updates to the compiler